api/src/index.js

```diff
@@ -29,3 +29,61 @@ app.use("/api", apiRouter);
 app.listen(process.env.PORT, () => {
   console.log(`API listening on port ${process.env.PORT}`);
 });
+//Future-meal
+app.get('/future-meals', async (req, res) => {
+  try {
+    const now = new Date();
+    const meals = await knex('Meal').where('when', '>', now);
```

Comment:
You named your table `Meal`, checkout this page for SQL table/column name conventions: [https://brainstation.io/learn/sql/naming-conventions](https://brainstation.io/learn/sql/naming-conventions)